scripts/get_ready_to_merge_prs.py

```diff
@@ -78,4 +76,11 @@ def parse_urls(data):
 
 
 if __name__ == "__main__":
-    get_ready_to_merge_prs()
+    if len(sys.argv) > 1:
+        org = sys.argv[1]
+        if org.lower() not in ORGS:
+            print(f"organization name {org} is not valid. Valid Orgs: {ORGS}")
+            sys.exit(1)
+        get_ready_to_merge_prs(org)
+    else:
+        print(f"{', '.join(ORGS)} are vallid organizations")
```

Comment:
What's the reason for this validation, if the code is meant to be re-usable, why limit it to just the two orgs?  Why not allow any value for the `org` argument and then just validate that the org exists and that the"user" running the script has sufficient access to to the org to accomplish their work?